docs/resources/trust_token.md

```diff
@@ -0,0 +1,57 @@
+# lxd_trust_token
+
+The `lxd_trust_token` resource allows you to create new tokens in the LXD trust store.
```

Comment:
I'm not sure the tokens live in the trust store. They are a way to get your cert into the trust store though.